### ISSUE TITLE
[BUG] Window Appears Off Screen if Second Monitor DCs

### ISSUE THREAD

**Issue by elibroftw** [tauri-apps/tauri-plugin-window-state#40]
If I open the app, move the app to a second screen, close the app, disconnect my monitor from the laptop, and then reopen the app, the app opens off screen.

The window should be opened as default if the last position is considered off screen. 

Thanks.

**Comment by amrbashir** [comment]
try with [fallback-to-primary](https://github.com/tauri-apps/tauri-plugin-window-state/tree/fallback-to-primary) branch

**Comment by elibroftw** [comment]
Buggier.
When the window (not fullscreened) is on the primary monitor, it will always open in the top left corner.
If the window is closed in the second monitor and reopened, it reopens in the second monitor fine.
If the window is closed in the second monitor and reopened when there is no second monitor, it opens off screen.

**Comment by amrbashir** [comment]
can you try again with the latest commit

**Comment by elibroftw** [comment]
Works perfecting thanks

**Comment by elibroftw** [comment]
Actually I think there's a regression. When the second screen is the primary display (I switched up the display port cables when I replugged the cables) and I resize the app on the second display (first screen), the resize is only respected on the primary display and not the extended/second. I'll try reproducing from my laptop tomorrow, but here is the app. 
https://github.com/elibroftw/google-keep-desktop-app
